src/debug/daccess/dacimpl.h

```diff
@@ -3904,7 +3904,8 @@ class EnumMethodInstances
 #define DAC_ENTER() \
     EnterCriticalSection(&g_dacCritSec); \
     ClrDataAccess* __prevDacImpl = g_dacImpl; \
-    g_dacImpl = this;
+    g_dacImpl = this; \
+    g_gcDacGlobals = &g_gc_dac_vars;
```

Comment:
Nevermind, this is from an older revision.

---
In reply to: [99724301](https://github.com/dotnet/coreclr/pull/9255#discussion_r99724301) [](ancestors = 99724301)